pyatlan/pkg/templates/logging_conf.jinja2

```diff
@@ -0,0 +1,48 @@
+[loggers]
```

Comment:
Strange to me that this is a per-package template, when there's nothing templated in it — I guess this is to workaround something in the logging (?)

Reply:
It's intended to be used by the custom package toolkit. I just made it a template in case we wanted to add any customization in the future.